<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomizerTabWidget</class>
 <widget class="QTabWidget" name="RandomizerTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RandomizerTabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="usesScrollButtons">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="levantTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/Levant16.png</normaloff>:/icons/Levant16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Levant</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="levantScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>796</width>
         <height>570</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="minionTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/Ocarina16.png</normaloff>:/icons/Ocarina16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Minion</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="minionScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="bossTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/FireBoss16.png</normaloff>:/icons/FireBoss16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Boss</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="bossScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>795</width>
         <height>570</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="forestTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/DragonflyKey16.png</normaloff>:/icons/DragonflyKey16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Forest</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_10">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="forestScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="equipmentTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/Garb16.png</normaloff>:/icons/Garb16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Equipment</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_9">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="equipmentScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="treasureTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/Bag16.png</normaloff>:/icons/Bag16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Treasure</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="treasureScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="shopTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/TwoSword16.png</normaloff>:/icons/TwoSword16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Shop</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_7">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="shopScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="miscTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/BlueCursor16.png</normaloff>:/icons/BlueCursor16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Misc</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_8">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="miscScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="addonsTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/Addons16.png</normaloff>:/icons/Addons16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Addons</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="addonsScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="fixesTab">
   <attribute name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/BetaLoading16.png</normaloff>:/icons/BetaLoading16.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Fixes</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="fixesScroll">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
